_data/projects/job-requirements.yml

```diff
@@ -8,5 +8,5 @@ upforgrabs:
   name: up for grabs
   link: https://github.com/crystal-dawn/job-requirements/labels/up%20for%20grabs
 stats:
-  issue-count: 2
+  issue-count: 0
```

Comment:
I don't believe this is the complete fix, because I don't see the label above listed on [the repository](https://github.com/crystal-dawn/job-requirements/labels):

<img width="364" src="https://user-images.githubusercontent.com/359239/66701081-77de8280-ecce-11e9-8b34-2b1b135f788e.png">

@crystal-dawn is there a label you would like to switch to now that `up for grabs` isn't listed?

Reply:
Thanks I opened a new issue with up for grabs. I made some changes earlier and didnt realize it affected the issue. Thanks for bringing it to my attention.